### PR TITLE
Story/tg 35

### PR DESCRIPTION
**Pull request by disist**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-frontend/pull/11" target="_blank">ReviewNinja

File: e2e/example_spec.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+describe('angularjs homepage', () => {
+		it('should greet the named user',() => {
+				browser.get('http://www.angularjs.org');
```
Comment (kosich) [inline e2e/example_spec.js]: !fix
remove example spec and add test, that, say, checks _our code_ that we have headline 'SmartHouse'

File: package.json [hunk 2]
```diff
@@ -70,7 +74,7 @@
     "angular2": "^2.0.0-beta.14",
     "codecov": "^1.0.1",
     "core-js": "^2.2.2",
-    "reflect-metadata": "^0.1.3",
+    "reflect-metadata": "^0.1.2",
```
Comment (kosich) [inline package.json]: why was this package version lowered?

Comment (ConstantineMarinsky) [inline package.json]: fixed

File: e2e/example_spec.js (hunk 1 above)
Comment (ConstantineMarinsky) [inline e2e/example_spec.js]: !fixed

File: e2e/dashboard_title_spec.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+describe('SmartHouse homepage', () => {
+    it('should check page title', () => {
+        browser.get('https://garage-it-smart-house.herokuapp.com/');
```
Comment (yurii-sorokin) [inline e2e/dashboard_title_spec.js]: !fix replace hardcoded url with configurable or relative url

Comment (yurii-sorokin) [inline e2e/dashboard_title_spec.js]: !fixed

**Comment by artem713** [comment]
Could you please squash some of your commits (duplicate commits, fix commits etc)

File: e2e/archive/device_list_spec.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+        browser.driver.manage().window().maximize();
+    });
+
+    it('should check page title', () => {
+        browser.get('https://garage-it-smart-house.herokuapp.com/');
+		
+		element(by.css(".router-link-active")).click();
+		
+		//var greeting = element(by.binding('yourName'));
+		//expect(greeting.getText()).toEqual('Hello Julie!');
```
Comment (artem713) [inline e2e/archive/device_list_spec.js]: I don't think we still need it

File: e2e/pages/home.page.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+var HomePage = function () {
+    
+    this.greetingHeader = $('main.container h1');
```
Comment (artem713) [inline e2e/pages/home.page.js:3]: I think it would be better to search elements by id or class name rather than tag.

Comment (kosich) [inline e2e/pages/home.page.js:3]: its a good practice to have specific attributes for such locators
for example, `data-e2e="layout::header__title"`

File: e2e/archive/device_list_spec.js (hunk 4 above)
Comment (dkondakov) [inline e2e/archive/device_list_spec.js]: !fixed

File: e2e/pages/home.page.js (hunk 5 above)
Comment (dkondakov) [inline e2e/pages/home.page.js:3]: Should I open a bug related to assigning ids/attributes for all valuable elements or mark this comment as fixed?

Comment (dkondakov) [inline e2e/pages/home.page.js:3]: !fixed

**Pull request merged**